app/scripts/components/common/map/style-generators/raster-timeseries.tsx

```diff
@@ -11,7 +11,7 @@ import {
 } from 'mapbox-gl';
```

Comment:
Please remind me:
I recall us having a discussion about whether or not to make the STAC layer data requests outside the layer component. Did we end up deciding to keep it this way because of how tightly coupled with the rendering these requests are?

Reply:
Yes, that is correct